src/main/java/de/tum/in/www1/artemis/repository/TeamRepository.java

```diff
@@ -20,6 +20,9 @@
 
     List<Team> findAllByExerciseCourseIdAndShortName(@Param("courseId") Long courseId, @Param("shortName") String shortName);
 
+    @Query(value = "select count(distinct team) from Team team where team.exercise.id = :#{#exerciseId}")
```

Comment:
```suggestion
    @Query("select count(distinct team) from Team team where team.exercise.id = :#{#exerciseId}")
```
Maybe that would fix the Codacy notice. I don't know how important it is though.